packages/fuel-indexer/src/ffi.rs

```diff
@@ -202,8 +232,21 @@ impl<'a> WasmArg<'a> {
             .exports
             .get_typed_function::<(u32, u32), ()>(store, "dealloc_fn")
             .expect("No dealloc fn");
-        dealloc_fn
-            .call(store, self.ptr, self.len)
-            .expect("Dealloc failed");
+        // Need to track whether metering is enabled or otherwise getting or setting points will panic
+        if self.metering_enabled {
+            let pts = match get_remaining_points(store, self.instance) {
+                MeteringPoints::Exhausted => 0,
+                MeteringPoints::Remaining(pts) => pts,
+            };
+            set_remaining_points(store, self.instance, 1_000_000);
```

Comment:
Is there some significance to the 1M number here? It seems like it may not be necessary for `dealloc_fn` as I can't see that costing too much; additionally, we set it immediately after the call. Also, is there a reason that we get the remaining points but don't use it until after the deallocation?

Reply:
I missed this earlier, but @lostman #NoMagicNumbers - very very rarely should be including raw values in production code :) - and if you see us doing this elsewhere in the repo bring it to our attention so we can correct it